src/api/dictionary.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+var htmlBodyValues = {
```

Comment:
this is dictionary with current body encoding rules(described in csl hld) , which mobile app uses right now and list of possible events with array of possible lables (labelids)(from csl hld too).